src/main/groovy/fr/inria/gforge/spoon/SpoonTask.groovy

```diff
@@ -40,7 +40,7 @@ class SpoonTask extends DefaultTask {
             addKey(params, '-x')
         }
         if (processors.size() != 0) {
-            addParam(params, '-p', processors.join(':'))
+            addParam(params, '-p', processors.join(File.separator))
```

Comment:
In the snippet of code you quote, it's `File.pathSeparator` which is used, not `File.separator`.

Reply:
Thank you. Correcting it now.